src/main/scala/scala/tools/partest/nest/Runner.scala

```diff
@@ -131,9 +131,7 @@ class Runner(val testFile: File, val suiteRunner: SuiteRunner, val nestUI: NestU
     )
 
     pushTranscript(args mkString " ")
-    val captured = StreamCapture(runCommand(args, logFile))
-    if (captured.result) genPass() else {
-      logFile appendAll captured.stderr
+    if (runCommand(args, logFile)) genPass() else {
```

Comment:
To answer the question in the commit message: there were two tickets, on the first someone comments that process `#>` doesn't do stderr. So `StreamCapture` came first. The proper fix using process logger came second.

That was exactly five years ago this past US Memorial Day weekend.

I said: In about five years, paulp will levy a commit, "Remove commented code from 2013." I still can't believe it's 2013. It is 2013, right?

You said: I don't recall having to enter 2-0-1-3 on a form this year. Usually I write down the previous year the first few times; after that my brain comes to grips with the annus novus.

It turns out there is no coming to grips.

https://github.com/scala/bug/issues/6289
https://github.com/scala/bug/issues/7003
https://github.com/scala/scala/pull/2591#commitcomment-3291466

Reply:
Thanks for the nostalgia trip :)